dsdl_compiler/README.md

```diff
@@ -0,0 +1,108 @@
+# UAVCAN DSDL compiler for the libcanard
+
+## Overview
+Libcanard_dsdlc is a tool to generate UAVCAN DSDL definitions to libcanard compatible message c-modules.
+
+Modules have: defines, enums, unions, structs, and encoding and decoding functions as defined in UAVCAN DSDL. Encoding and decoding functions use the decode and encode functions of libcanard for bit packing / unpacking.
+
+In c there is no namespace, so all the generated #define and function names are long having full folder path included. This is made to prevent the collision with each other and to the rest of the system.
+
+## Install & Integration
+To get libcanard from the git, make sure all the submodules are fetched too.
+
+`git submodules update --init --recursive`
+
+### Generating files
+`python3 libcanard_dsdlc --outdir <outdir> <dsdl-definition-uavcan-folder>`
+
+NOTE: If python2 is used, monotonic library is needed.
```

Comment:
monotonic is named, but the pip command installs monolith, which one is the correct one ?

Reply:
`monotonic` is correct, but it's not really required. Python 3 is pretty much ubiquitous at this point, and even if Python 2 is used without the `monotonic` package, pyuavcan will fall back to using wall time instead of monotonic time. This is perfectly acceptable for the compiler because it is not a real-time application.